lib/engine/game/g_1854/step/merge_minors.rb

```diff
@@ -0,0 +1,114 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/base'
+
+module Engine
+  module Game
+    module G1854
+      module Step
+        class MergeMinors < Engine::Step::Base
+          def actions(entity)
+            return [] if entity.corporation? && !entity.minor?
+            return [] if !entity.minor? || entity != current_entity #  || @game.done_this_round[entity]
+            return [] unless @game.mergeable?(entity)
+            return [] if target_corporations.empty?
+            return [] if @merge_target
+
+            actions = []
+            actions << 'merge' if @game.minor_mergers_allowed?
+            actions << 'pass' if @game.minor_mergers_allowed? && !@game.minor_mergers_required?
+            actions
+          end
+
+          def blocking?
+            @game.mergeable?(current_entity)
+          end
+
+          def active?
+            return false if @game.open_minors.empty?
+
+            @game.minor_mergers_allowed? || @game.minor_mergers_required?
+          end
+
+          def setup
+            @mergeable_ = {}
+            @merging = nil
+            @merge_target = nil
+          end
+
+          def auto_actions(entity)
+            return super if @merging
+
+            return [Engine::Action::Pass.new(entity)] if mergeable_candidates(entity).empty?
+
+            super
+          end
+
+          def merge_name(entity = nil)
+            return "Form #{entity&.name}" if @merging
+
+            "Merge with #{entity&.name}"
+          end
+
+          # corps available as a merge target
+          def target_corporations
+            @game.corporations.select { |c| @game.merge_target?(c) }
```

Comment:
What does the `@game.merge_target?` method do?

Reply:
https://github.com/andrewzwicky/18xx/blob/80f2537d5a4c495dc9f4612e7f5b6cb913236bbb/lib/engine/game/g_1854/game.rb#L283-L285